src/rgw/rgw_rest_s3.cc

```diff
@@ -408,14 +417,28 @@ void RGWGetUsage_ObjStore_S3::send_response()
        formatter->dump_int("SuccessfulOps", total_usage.successful_ops);
        formatter->close_section(); // total 
        formatter->close_section(); // user
-     } 
-     formatter->dump_int("TotalBytes", header.stats.total_bytes);
```

Comment:
I'm not sure whether changing the json response here is a good idea, there might be other consumers who are built on top the rest api. For eg. the openstack ceilometer rgw plugin iirc used the "Summary" bits for quite sometime.. not sure what the state is right now